docs/sources/alerting/manage-notifications/webhook-notifier.md

```diff
@@ -110,12 +110,15 @@ Example JSON body:
 | annotations  | object | Annotations that are part of this alert, map of string keys to string values       |
 | startsAt     | string | Start time of the alert                                                            |
 | endsAt       | string | End time of the alert, default value when not resolved is `0001-01-01T00:00:00Z`   |
-| valueString  | string | Values that triggered the current status                                           |
+| valueString  | string | **Will be deprecated in Grafana 10**                                               |  
+| values       | object | Values that triggered the current status                                           |
 | generatorURL | string | URL of the alert rule in the Grafana UI                                            |
 | fingerprint  | string | The labels fingerprint, alarms with the same labels will have the same fingerprint |
 | silenceURL   | string | URL to silence the alert rule in the Grafana UI                                    |
 | dashboardURL | string | **Will be deprecated soon**                                                        |
 | panelURL     | string | **Will be deprecated soon**                                                        |
+| imageURL     | string | URL of the screenshots of the panel alert rule assigned to notifications           |
+| embeddedImage| string | embedded screenshots of the panel alert rule assigned to notifications             |
```

Comment:
embeddedImage are not populated in webhook. it is used only by email integration

Reply:
Ok, I will remove it.

Reply:
Done.